### ISSUE TITLE
Is there any option to open info window by default when the map is loaded ? 

### ISSUE THREAD

**Issue by mohamednagy** [bradcornford/Googlmapper#72]

**Comment by bradcornford** [comment]
Hi,

Yes, this is possible. See an example below:

```php
Mapper::map(53.3, -1.4, ['zoom' => 15, 'center' => false, 'marker' => false, 'type' => 'HYBRID', 'eventAfterLoad' => 'google.maps.event.trigger(maps[0].markers[1], "click");']);
Mapper::informationWindow(53.4, -1.3, 'Content');
```

I will look at implementing a solution for this in the near future.

**Comment by bradcornford** [comment]
Hi, 

I have now implemented this feature in versions v2.14.0/v1.10.0.

Here is the usage:

```php
Mapper::informationWindow(53.3, -1.4, 'Content', ['title' => 'This is open by default', 'open' => true]);
```

**Comment by mohamednagy** [comment]
+1 thank you
